src/opnsense/scripts/shell/setaddr.php

```diff
@@ -542,6 +542,12 @@ function console_configure_dhcpd($version = 4)
         unset($config['system']['webgui']['noantilockout']);
         $restart_webgui = true;
     }
+    if (isset($config['interfaces']['lan'])) {
```

Comment:
What this code tries to achieve is go back to the factory defaults. We are not the LAN interface... we are any interface here and reverting to defaults attempts to restore GUI access, but not for a definitive "lan" interface. This script allows the user to select the interface to set up prior (and asking for anti-lockout there might be more sensible).

Reply:
(more to your original point the 'lan' key could have been repurposed as the WAN interface in the meantime)